### DISCUSSION TITLE
Automatic screenshots via GitHub Actions.

### DISCUSSION THREAD

**Discussion by dennykorsukewitz** [karldaeubel/PenAndPaperFloorplanner#27] [Ideas]
I like the idea that GitHub creates the latest screenshots for us.

I'll take a look at that topic.

🚀 🍻

**Comment by karldaeubel** [comment]
Sounds like a great idea!

Is it also possible to load floorplans and zoom/translate the view with this approach?

**Comment by dennykorsukewitz** [comment]
That's what I'm working on at the moment.
I already have the images for the different pages.
But I still need a js file that automatically uploads the example file after the page has been loaded.

Maybe somehing linke that...
https://pqina.nl/blog/set-value-to-file-input/#updating-the-file-input-files-property

**Reply by dennykorsukewitz** [reply]
I've moved on. maybe I'll make it tomorrow.

**Comment by dennykorsukewitz** [comment]
# BOOM https://github.com/karldaeubel/PenAndPaperFloorplanner/pull/28

![fzJ3](https://github.com/karldaeubel/PenAndPaperFloorplanner/assets/3249687/fe3d4d89-6ca4-4b1e-b5fd-186810e35c0d)

**Comment by karldaeubel** [comment]
It works great, thanks :) One more issue, the action has difficulties commiting when merging pull requests. You got any idea why that is?

**Reply by dennykorsukewitz** [reply]
What is the problem?
That it does not run?
Error?

**Reply by karldaeubel** [reply]
This is the error message, he has no access right to push/commit:

Screenshot
Error: Pushing to https://github.com/karldaeubel/PenAndPaperFloorplanner
remote: Permission to karldaeubel/PenAndPaperFloorplanner.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/karldaeubel/PenAndPaperFloorplanner/': The requested URL returned error: 403

**Reply by dennykorsukewitz** [reply]
Permission  denied:

did you do that?

![image](https://github.com/karldaeubel/PenAndPaperFloorplanner/assets/3249687/c4a4d61b-d14e-4934-8c3e-f8151534528d)

**Reply by karldaeubel** [reply]
Yeah, it also works when I trigger the workflow manually.

**Reply by dennykorsukewitz** [reply]
https://github.com/karldaeubel/PenAndPaperFloorplanner/pull/33

**Comment by karldaeubel** [comment]
Any ideas why the screenshots can not display certain unicode characters? Especially arrows, e.g. ⮈ or 🡲.

**Reply by karldaeubel** [reply]
I even extended the fallback fonts and it still doesn't work :thinking: 

`font: normal 15px "Segoe UI", Arial, Helvetica, sans-serif;`

**Reply by dennykorsukewitz** [reply]
fixed?
